truffleruby/src/main/ruby/core/truffle/interop.rb

```diff
@@ -66,6 +66,9 @@ class Foreign
 
     end
 
+    module Java
+      # Home for low-level Java interop functions - currently implemented in the interpreter.
+    end
```

Comment:
I don't think we need this. It's only small but every little thing slows down startup.